TumeKyouen/sample.swift

```diff
@@ -0,0 +1,10 @@
+//
+//  sample.swift
+//  TumeKyouen
+//
+//  Created by noboru ishikura on 2015/12/10.
+//  Copyright © 2015年 noboru. All rights reserved.
+//
+
+import Foundation
+
```

Comment:
ERROR: [Warning][trailing_newline] Trailing Newline
Files should have a single trailing newline.